extras/benchmarking/benchmark_sync_v2.sh

```diff
@@ -0,0 +1,45 @@
+N_BLOCKS=100
```

Comment:
Change this back to 20k. I only changed it to 100 to run the CI quickly and generate a fake result to demonstrate Bencher plots, as the real plot will only exist after this PR is merged to master.